src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -11,7 +11,18 @@ namespace System.Diagnostics
     // another version of this partial class with the public visibility 
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if MONO
```

Comment:
That would definitely makes sense, but it's a bit more verbose, which is not necessarily what you want, as you would still need to have `Debug.NewLine` (which would be `NewLine = Unix.NewLine` in `Debug.Unix.cs` and `NewLine = Windows.NewLine` in `Debug.Windows.cs`).